examples/chip-tool/commands/clusters/ModelCommand.cpp

```diff
@@ -33,7 +33,9 @@ void ModelCommand::OnDeviceConnectedFn(void * context, ChipDevice * device)
 {
     ModelCommand * command = reinterpret_cast<ModelCommand *>(context);
     VerifyOrReturn(command != nullptr, ChipLogError(chipTool, "OnDeviceConnectedFn: context is null"));
```

Comment:
Does the command exit status need to be set here too?

Reply:
I did thing the same thing at first,  but the issue is that we don't have a "command" there :D